build.gradle

```diff
@@ -31,6 +31,17 @@ def getSubprojectVersion(project, version) {
 	}
 }
 
+def getBranch(){
```

Comment:
These changes arent right, seems to be targeting the wrong branch. (`1.14` is correct)